src/quic/node_quic_socket.cc

```diff
@@ -658,15 +649,15 @@ bool QuicSocket::SendStatelessReset(
   if (pktlen < kMinStatelessResetLen)
     return false;
 
-  GenerateResetToken(token, reset_token_secret_, cid);
+  StatelessResetToken token(reset_token_secret_, cid);
   EntropySource(random, kRandlen);
 
   auto packet = QuicPacket::Create("stateless reset", pktlen);
   ssize_t nwrite =
       ngtcp2_pkt_write_stateless_reset(
         reinterpret_cast<uint8_t*>(packet->data()),
         NGTCP2_MAX_PKTLEN_IPV4,
-        token,
+        const_cast<uint8_t*>(token.data()),
```

Comment:
This is an ngtctp2 bug, right? Afaict `ngtcp2_pkt_write_stateless_reset()` should use `const uint8_t*` for `stateless_reset_token` and `rand`?

Reply:
I believe so yes

Reply:
https://github.com/ngtcp2/ngtcp2/pull/210